code/CMakeLists.txt

```diff
@@ -923,7 +923,7 @@ IF(ASSIMP_HUNTER_ENABLED)
   hunter_add_package(utf8)
   find_package(utf8cpp CONFIG REQUIRED)
 ELSE()
-  # utf8 is header-only, so Assimp doesn't need to do anything.
+  INCLUDE_DIRECTORIES("../contrib/utf8cpp/source")
```

Comment:
i agree that it is much better to just fix the include path, rather than including different files. based on context as is done now: https://github.com/assimp/assimp/blob/020554e213365e009f5455ae3572bb0ae10026eb/code/Common/BaseImporter.cpp#L360-L364

Reply:
obviously this PR does not (yet) change the actual includes (e.g. in `BaseImporter.cpp`.

after adding the include_directories, it could be as simple as:

```C
#include <utf8.h>
```